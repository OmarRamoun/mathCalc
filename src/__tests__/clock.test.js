import renderer from 'react-test-renderer';
import Clock from '../components/Clock';

test('Render components', () => {
    const tree = renderer.create(<Clock />);
    expect(tree).toMatchSnapshot();
});