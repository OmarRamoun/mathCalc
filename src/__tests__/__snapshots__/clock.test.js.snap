// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render components 1`] = `
<div
  className="col-md-9"
>
  <div
    className="row text-end text-white"
  >
    <p>
      17:28:25
    </p>
  </div>
</div>
`;
